src/app/detail/ui/Sidebar.tsx

```diff
@@ -88,7 +98,7 @@ export const Sidebar = ({
         borderLeft: (theme) => `1px solid ${theme.color.borders.border2}`,
         height: '100vh',
         display: showSidebar ? 'block' : 'none',
-        width: matches && showSidebar ? '100vw' : '25vw',
+        width: isMobile && showSidebar ? '100vw' : '25vw',
```

Comment:
```suggestion
        width: showSidebar ? '100vw' : '25vw',
        md: {
         width '25vw'
        }
```

Something like this should be possible and would give you much better result as you would not always have window.width defined.